src/lib/pages/workflow-query.svelte

```diff
@@ -45,6 +51,33 @@
   let queryResult: Promise<ParsedQuery>;
   let encodePayloadResult: Promise<Payloads>;
 
+  onMount(() => {
+    if (!$workflowRun?.metadata) {
```

Comment:
Looks like `$workflowRun` should always be defined and has an initial default. 
```suggestion
    if (!$workflowRun.metadata) {
```

Reply:
👍